File: src/BreakfastProvider.Api/Models/Responses/ChefNoteResponse.cs
namespace BreakfastProvider.Api.Models.Responses;

public class ChefNoteResponse
{
    public string NoteId { get; set; } = string.Empty;
    public string RecipeName { get; set; } = string.Empty;
    public string ChefName { get; set; } = string.Empty;
    public string NoteText { get; set; } = string.Empty;
    public string Category { get; set; } = string.Empty;
    public DateTime CreatedAt { get; set; }
    public DateTime? UpdatedAt { get; set; }
}

File: java/breakfast-provider-app/src/main/java/io/lemonlion/breakfast/model/response/ChefNoteResponse.java
package io.lemonlion.breakfast.model.response;

import java.time.Instant;

/** Twin of C# {@code ChefNoteResponse}. */
public record ChefNoteResponse(
        String noteId,
        String recipeName,
        String chefName,
        String noteText,
        String category,
        Instant createdAt,
        Instant updatedAt) {
}
